Chaincase.Common/Global.cs

```diff
@@ -199,7 +199,7 @@ public async Task InitializeNoWalletAsync(CancellationToken cancellationToken =
 
                 #region SynchronizerInitialization
 
-                var requestInterval = TimeSpan.FromSeconds(30);
+                var requestInterval = TimeSpan.FromSeconds(3);
```

Comment:
Problem: connection confirmation timeout must be reduced to fit CoinJoin within 30s window, however this background confirmation timeout also dictates how often it expects the client Alice to check in. Less often and we get dropped registration. dropping registration status during InputRegistration is more of an issue with the UI than a practical one since coinjoin is triggered by a push notification. Polling this often may overwhelm the coordinator

Reply:
https://github.com/chaincase-app/Chaincase/issues/202